app/assets/javascripts/oxalis/model/bucket_data_handling/prefetch_strategy_plane.js

```diff
@@ -21,20 +21,8 @@ export class AbstractPrefetchStrategy {
   u: number;
   v: number;
 
-  forContentType(contentTypes: { skeleton: boolean, volume: boolean }): boolean {
-    if (_.isEmpty(this.contentTypes)) {
```

Comment:
I think we still need this check (I misinterpreted it while talking to you earlier) as the arbitrary prefetch strategy doesn't define any content types. If a prefetch strategy doesn't define any content types, it should be used for all content types - at least that was the original behavior.
Alternatively, we could also add `contentTypes = ["skeleton", "readOnly"];` to the arbitrary prefetch strategy.

Reply:
Very good point! I re-added the missing check and also renamed the variables a bit, since our initial misinterpretation was probably (at least partly) due to the confusing `contentTypes` and `this.contentTypes` naming.